cmd/storage.go

```diff
@@ -83,6 +84,23 @@ func printBlobbers(nodes []*sdk.Blobber) {
 	}
 }
 
+func filterActiveBlobbers(blobbers []*sdk.Blobber) []*sdk.Blobber {
+
+	result := []*sdk.Blobber{}
+
+	for i := range blobbers {
+		if time.Now().Unix()*1000-int64(blobbers[i].LastHealthCheck*1000) < 3600000 {
+			result = append(result, blobbers[i])
+
+		} else {
+			fmt.Println("Inactive Blobber: ", blobbers[i].BaseURL)
+		}
+
+	}
+
+	return result
+}
+
```

Comment:
let's drop excessive code. at first is there need to multiply by1000 the both sides of the comparison ;)? also lets apply existing utility methods for the Timestamp struct. and finally, named return param would be really helpful in this very function. so get:

```
func filterActiveBlobbers(blobbers []*sdk.Blobber) (activeBlobbers []*sdk.Blobber) {
    for i := range blobbers {
        if blobbers[i].LastHealthCheck.Within(60 * 60) {
            activeBlobbers = append(activeBlobbers, blobbers[i])
        }
    }
   return
}
```

PS. ideally such function should be added, as a method of the blobber list struct, right to the SDK.

Reply:
Yes this funciton must be added  in SDK layer.

Reply:
What should we do then, make updates in SDK and then changes in zboxcli for merge?
Or for now implement funciton in cli and after making updates in SDK again another merge?
@andrenerd

Reply:
Checked the sdk codebase, there is no "[]*Blobber" struct there.
Well, in such case, lets add the function right here (zbocli), not extending sdk for now.